static.config.js

```diff
@@ -569,6 +572,33 @@ const getEventPageData = async (id, client) => {
   return { eventPage: eventPage };
 };
 
+const getAllEvents = async client => {
+  // Is this how we want to handle it? 
+  const date_now = moment().format('YYYY-MM-DD')
```

Comment:
considering this also being toggled as Austin timezone, im trying to figure out where netlify's timezone is

Reply:
you can just force it always be CST
- something like this
- `moment.utc('07-18-2013 16:10:11', 'MM-DD-YYYY HH:mm').tz('America/Chicago').format("YYYY-MM-DD HH:mm");
`
- I don't think there is an Austin specific as the timezone is anchored to Chicago i guess. (Why does Chicago think it's so special!) But, that accounts for daylight saving too, which is good!

Reply:
Definitely coerce it into CT. We don't want our build output to be contingent on the timezone of our build server.

Reply:
also thank you @robbobfrh84 for doing the heavy lifting and finding how to do it!